src/main/java/it/gov/pagopa/pu/workflow/wf/classification/iuf/IufClassificationWFClient.java

```diff
@@ -37,7 +37,7 @@ public String notifyTreasury(IufClassificationNotifyTreasurySignalDTO signalDTO)
   }
 
   public String notifyPaymentsReporting(IufClassificationNotifyPaymentsReportingSignalDTO signalDTO) {
-    String workflowId = generateWorkflowId(signalDTO.getOrganizationId(), signalDTO.getIuf());
+    String workflowId = generateWorkflowId(signalDTO.getTransfers().getFirst().getOrgId(), signalDTO.getIuf());
```

Comment:
if we store the organizationId on the signalDTO class, here we could read it directly from the DTO itself (as it was before)
```suggestion
    String workflowId = generateWorkflowId(signalDTO.getOrganizationId(), signalDTO.getIuf());
```